framework/src/main/java/org/tron/core/Wallet.java

```diff
@@ -1740,6 +1722,21 @@ public TransactionCapsule createShieldedTransactionWithoutSpendAuthSig(
     return transactionCapsule;
 
   }
+  
+  private void ShieldedTransactionOutput (List<ReceiveNote> shieldedReceives,
```

Comment:
Class method naming style matters.
should be a verbal phrase. and should be in `lowerCamelCase`

Reply:
done, thanks